helpers/compile/build.ts

```diff
@@ -168,10 +168,18 @@ export async function build(options: BuildOptions[]) {
 
   return transduce.async(
     createBuildOptions(options),
-    pipe.async(computeOptions, addExtensionFormat, addDefaultOutDir, executeEsBuild),
+    pipe.async(computeOptions, logStartBuild, addExtensionFormat, addDefaultOutDir, executeEsBuild),
   )
 }
 
+/**
+ * Prints a message every time a new bundle is built
+ */
+function logStartBuild(options: BuildOptions): BuildOptions {
+  console.log(`Building ${options.name} as ${options.format ?? 'cjs'}...`)
+  return options
+}
```

Comment:
Added some progress logs because client takes a while to build, especially with all the new bundles, so it's nice to have some visual feedback.